pkg/scannerv4/mappers/mappers.go

```diff
@@ -449,7 +477,10 @@ func toProtoV4VulnerabilitiesMap(ctx context.Context, vulns map[string]*claircor
 // issuedTime attempts to return the issued time for the vulnerability.
 // If ccTime is non-zero, that time is preferred. Otherwise, if the nvdTime is populated, then use that.
 // Otherwise, return nil.
-func issuedTime(ccTime time.Time, nvdTime string) *timestamppb.Timestamp {
+func issuedTime(advisoryTime, ccTime time.Time, nvdTime string) *timestamppb.Timestamp {
+	if !advisoryTime.IsZero() {
+		return protocompat.ConvertTimeToTimestampOrNil(&advisoryTime)
+	}
```

Comment:
It would be cleaner and easier to track if you continue to accept only `ccTime` (maybe renamed) based on `advisoryExists` when calling. Consider this a nitpick on style and code organization.